React Toolbar content and feedback updates (#220)
patternfly/training-scenarios: react-components/toolbar-filter/step4.md

> One particular type of Data toolbar group is called the toggle group. A `DataToolbarToggleGroup` can be used to collapse a set of items into an overlay panel at a certain breakpoint. This allows complex toolbars with multiple items and groups of items to be responsive. 

Reviewer: sentence case for data toolbar

A toggle group is useful for containing filter controls, for example. When the toolbar responds to adapt to a mobile viewport, the contents contained in a toggle group will collapse into an overlay panel that can be toggled by clicking the Filter icon.

Using a `DataToolbarToggleGroup` requires both a `toggleIcon` and `breakpoint` property. The `toggleIcon` is a ReactNode which is displayed instead of the children of `DataToolbarToggleGroup` once the viewport width shrinks below the passed in breakpoint.

> 1) **Locate both the `DataToolbarItem` with `variant="search-filter"` and the `DataToolbarGroup` with `variant="filter-group"`:**

Reviewer: ```suggestion
1) **Locate both the `DataToolbarItem` with `variant="search-filter"` and the `DataToolbarGroup` with `variant="filter-group"`.**
```

**Hint:** Both the `DataToolbarItem` and `DataToolbarGroup` to locate are siblings of each other.

<pre class="file">
&lt;DataToolbarItem variant=&quot;search-filter&quot;&gt;
  &lt;Input value=&quot;&quot; id=&quot;step2Input&quot; ariaLabel=&quot;Step 2 input&quot; /&gt;
&lt;/DataToolbarItem&gt;
&lt;DataToolbarGroup variant=&quot;filter-group&quot;&gt;
  &lt;DataToolbarItem&gt;
    &lt;CheckboxSelect
      onSelect={this.onSelect}
      type=&quot;Status&quot;
      selections={filters.status}
      options={statusOptions}
    /&gt;
  &lt;/DataToolbarItem&gt;
  &lt;DataToolbarItem&gt;
    &lt;CheckboxSelect
      onSelect={this.onSelect}
      type=&quot;Risk&quot;
      selections={filters.risk}
      options={riskOptions}
    /&gt;
  &lt;/DataToolbarItem&gt;
&lt;/DataToolbarGroup&gt;
</pre>

> 2) **Wrap both the located `DataToolbarItem` and `DataToolbarGroup` in the same `DataToolbarToggleGroup`**

Reviewer: ```suggestion
2) **Wrap the `DataToolbarItem` and `DataToolbarGroup` located in step 1, in a `DataToolbarToggleGroup`**
```

> 3) **Add the `toggleIcon` property to the `DataToolbarToggleGroup` component. Its value should be `{<FilterIcon />}`**

Reviewer: Split this step up, and move the second sentence to a new line.

```suggestion
3) **Add the `toggleIcon` property to the `DataToolbarToggleGroup` component. **

The value of the `toggleIcon` property should be `{<FilterIcon />}`
```

> 4) **Add the `breakpoint` property to the `DataToolbarToggleGroup` component. Its value should be `"xl"`**  

Reviewer: ```suggestion
4) **Add the `breakpoint` property to the `DataToolbarToggleGroup` component.** 

The value of the `breakpoint` property should be `"xl"`  
```

      /&gt;
    &lt;/DataToolbarItem&gt;
  &lt;/DataToolbarGroup&gt;
&lt;/DataToolbarToggleGroup&gt;
</pre>

> 5) **Test changing viewport widths and breakpoints**

Reviewer: ```suggestion
5) **Test the responsiveness of the toolbar**
```
